packages/airnode-validator/templates/0.4/config.json

```diff
@@ -135,16 +135,14 @@
         ],
         "__optional": {
           "baseFeeMultiplier": {
-            "__type": "string",
-            "__regexp": "^[0-9]$",
+            "__type": "number",
             "__catch": {
               "__message": "__fullPath: Base Fee multiplier must be a whole number"
             }
           },
           "priorityFee": {
             "value": {
-              "__type": "string",
-              "__regexp": "^\\d*\\.?\\d*$",
```

Comment:
I simply removed the `regexp` since Zod has more options for numbers that can be used.